elements/pfe-codeblock/src/pfe-codeblock-material-dark.scss

```
@@ -0,0 +1,223 @@
+@import "../../pfe-sass/pfe-sass";
+
+//Define custom colors
+$custom-red: #FF6666;
+$custom-orange: #FE9A8C; //(red, #eee, etc.)
+$custom-yellow: #FFCB6B; //(height, width, cx, cy, etc.)
+$custom-green: #A5E844;
+$custom-teal: #80CBC4;
+$custom-blue: #73BCF7;
+$custom-purple: #C792EA;
+$custom-dark-gray: #B8BBBE; // (numbers and divider)
+$custom-gray: #B8BBBE; // (text)
+$custom-white: #FFF;
+$custom-black: #2f2f2f;
+$custom-hilight-bg: #363636;
+
+:host([pfe-c-codeblock-theme="dark"]) {
```

```suggestion
:host([codeblock-theme="dark"]) {
```

No need to prefix attributes anymore, 🥳

---

I tried this suggestion but the styles were not applied if I removed pfe-